checker/check-request-property-enum-value-updated.go

```diff
@@ -28,24 +28,38 @@ func RequestPropertyEnumValueRemovedCheck(diffReport *diff.Diff, operationsSourc
 				CheckModifiedPropertiesDiff(
 					mediaTypeDiff.SchemaDiff,
 					func(propertyPath string, propertyName string, propertyDiff *diff.SchemaDiff, parent *diff.SchemaDiff) {
-						enumDiff := mediaTypeDiff.SchemaDiff.EnumDiff
-						if enumDiff == nil || enumDiff.Deleted == nil {
-							return
-						}
-						if propertyDiff.Revision.Value.ReadOnly {
+						enumDiff := propertyDiff.EnumDiff
+						if enumDiff == nil {
 							return
 						}
+
 						for _, enumVal := range enumDiff.Deleted {
+							level := ERR
+							if propertyDiff.Revision.Value.ReadOnly {
+								level = INFO
+							}
 							result = append(result, ApiChange{
 								Id:          "request-property-enum-value-removed",
-								Level:       ERR,
+								Level:       level,
 								Text:        fmt.Sprintf(config.i18n("request-property-enum-value-removed"), enumVal, ColorizedValue(propertyFullName(propertyPath, propertyName))),
 								Operation:   operation,
 								OperationId: operationItem.Revision.OperationID,
 								Path:        path,
 								Source:      source,
 							})
 						}
+
+						for _, enumVal := range enumDiff.Added {
+							result = append(result, ApiChange{
+								Id:          "request-property-enum-value-added",
+								Level:       INFO,
+								Text:        fmt.Sprintf(config.i18n("request-property-enum-value-added"), enumVal, ColorizedValue(propertyFullName(propertyPath, propertyName))),
```

Comment:
Let's use ColorizedValue(enumVal) instead of enumVal

Reply:
since it's a new one I can do that here